apps/system/js/app_install_manager.js

```diff
@@ -243,22 +242,28 @@
       var id = detail.id;
 
       if (manifest.size) {
+        this.size.removeAttribute('data-l10n-id');
         this.size.textContent = this.humanizeSize(manifest.size);
       } else {
-        this.size.textContent = _('size-unknown');
+        this.size.setAttribute('data-l10n-id', 'size-unknown');
```

Comment:
L10n.js uses `MutationObserver` to observe any changes to `data-l10n-id` and `data-l10n-args` and translates affected nodes.